WalletWasabi.Tests/RegTests.cs

```diff
@@ -1898,19 +1898,19 @@ public async Task CcjTestsAsync()
 				byte[] dummySignature = new byte[65];
 
 				inputsRequest.BlindedOutputScripts = Enumerable.Range(0, round.MixingLevels.Count() + 1).Select(x => uint256.One);
-				inputsRequest.ChangeOutputAddress = new Key().PubKey.GetAddress(network);
+				inputsRequest.ChangeOutputAddress = new Key().PubKey.GetAddress(ScriptPubKeyType.Segwit, network);
```

Comment:
Q: what is more clear?
```c#
new Key().PubKey.GetAddress(ScriptPubKeyType.Segwit, network);
```
or,
```c#
new Key().PubKey.GetSegwitAddress(network);
```
or,
```c#
new Key().PubKey.WitHash.GetAddress(network);
```

IMO the second one looks better.

Reply:
I agree, but too lazy to mess with it. Feel free to make a PR if you want. Btw this part of nbitcoin is a mess: https://github.com/MetacoSA/NBitcoin/issues/286